src/App.tsx

```diff
@@ -1,57 +1,27 @@
-import React from 'react';
+import React, { useState } from 'react';
 import './App.scss';
+import { Autocomplete } from './Autocomplete';
```

Comment:
you haven't added this component to git

make `git add .`, commit, and push to make review possible. Then re-request review